GANDLF/training_loop.py

```diff
@@ -139,7 +139,10 @@ def train_network(model, train_dataloader, optimizer, params):
     # automatic mixed precision - https://pytorch.org/docs/stable/amp.html
     if params["model"]["amp"]:
         print("Using Automatic mixed precision", flush=True)
-        scaler = GradScaler()
+        if params["clip_grad"] is not None:
+            scaler = GradScaler()
+        else:
+            scaler = torch.cuda.amp.GradScaler()
```

Comment:
The GradScaler handles this automatically, this is not needed anymore. Please fix this.

Reply:
Actually, if that's the case, we should make a minor change for the if-loop later on, as well. Thanks, fixing it now.

Reply:
Check in #89